beater/server.go

```diff
@@ -80,7 +80,7 @@ type handler func(w http.ResponseWriter, r *http.Request)
 
 func createHandler(p processor.Processor, config Config, publish successCallback) handler {
 	return func(w http.ResponseWriter, r *http.Request) {
-		logp.Debug("handler", "Request - URI: %s ; Method: %s", r.RequestURI, r.Method)
+		logp.Debug("handler", "Request - URI: %s ; Method: %s; Content Length: %d", r.RequestURI, r.Method, r.ContentLength)
```

Comment:
Changed. This reminds me that we should start introducing metrics in our code that we can later use for monitoring. Will create a github issue.